games-simulation/simutrans/simutrans-nightly.recipe

```diff
@@ -9,18 +9,17 @@ changes which make your savegames unreadable by any official version, or \
 bugs that will destroy your game, or not even run. They are for bug testing \
 only."
 HOMEPAGE="http://www.simutrans.com"
-SOURCE_URI="svn://tron.yamagi.org/simutrans/simutrans/trunk \
-	--username=anon --password="
+SOURCE_URI="svn://servers.simutrans.org/simutrans/trunk"
 SOURCE_FILENAME="simutrans-$portVersion"
 SOURCE_DIR="trunk"
 COPYRIGHT="1997-2004 Hj. Malthaner
-	2005-2015 The Simutrans Team"
+	2005-2017 The Simutrans Team"
 LICENSE="Artistic"
-REVISION="2"
+REVISION="3"
 ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 PROVIDES="
-	simutrans$secondaryArchSuffix = $portVersion
+	simutrans$secondaryArchSuffix = $portVersion compat >= 120.2
```

Comment:
The ~ is used for pre-releases (alpha, beta, etc) and is "before" the matching release. So 1.2.1~something < 1.2.1. In other cases, minor releases and the like, are "after" the matching release. So, 1.2.1 > 1.2.
It seems that 1.2 == 1.2.0.

See "version strings" in this document for some extra info:
https://dev.haiku-os.org/wiki/PackageManagement/BuildingPackages